examples/pip_parse/BUILD

```diff
@@ -55,16 +55,6 @@ alias(
     actual = entry_point("yamllint"),
 )
 
-# The requirements.in file can be checked in to the source tree or it can be
-# generated. Pretend that we do some generating of the file. For this example,
-# the "template" is already the file we want.
-genrule(
```

Comment:
this was just dead code, so this is a drive-by cleanup right?

Reply:
No, it was not dead code. But I don't think we should be testing the generated requirements feature in the main `pip_parse` example. This is why I set up a more comprehensive test elsewhere.